sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -52,7 +52,7 @@ class TextAnalyticsClient(TextAnalyticsClientBase):
     https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview
 
     :param str endpoint: Supported Cognitive Services or Text Analytics resource
-        endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).
+        endpoints (protocol and hostname, for example: https://westus2.API.cognitive.microsoft.com).
```

Comment:
probably don't want to capitalize API here

Reply:
oh rip, thanks should've been more careful